pheweb/serve/server.py

```diff
@@ -256,11 +262,21 @@ def region_page(phenocode, region):
         pheno = phenos[phenocode]
     except KeyError:
         die("Sorry, I couldn't find the phewas code {!r}".format(phenocode))
+    chr_se = region.split(':')
+    chrom = chr_se[0]
+    if finemapping_dao is not None:
+        start_end = jeeves.get_max_finemapped_region(phenocode, chrom, chr_se[1].split('-')[0], chr_se[1].split('-')[1])
```

Comment:
make the use of jeeves/dao consistent. You can either call dao directly as in the next line or move next line to jeeves.